queue-staging-premium.tf

```diff
@@ -0,0 +1,116 @@
+locals {
+  staging_resource_count_premium = var.env == "aat" ? "1" : "0"
+}
+
+module "envelopes-staging-queue-premium" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=DTSPO-6371_azurerm_upgrade"
+  name                = "envelopes-staging"
+  namespace_name      = module.queue-namespace-premium.name
+  resource_group_name = azurerm_resource_group.rg.name
+
+  requires_duplicate_detection            = "true"
+  duplicate_detection_history_time_window = "PT59M"
+  lock_duration                           = "PT5M"
+  max_delivery_count                      = var.envelope_queue_max_delivery_count
+}
+
+module "processed-envelopes-staging-queue-premium" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=DTSPO-6371_azurerm_upgrade"
+  name                = "processed-envelopes-staging"
+  namespace_name      = module.queue-namespace-premium.name
+  resource_group_name = azurerm_resource_group.rg.name
+  lock_duration       = "PT5M"
+}
+
+module "payments-staging-queue-premium" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=DTSPO-6371_azurerm_upgrade"
+  name                = "payments-staging"
+  namespace_name      = module.queue-namespace-premium.name
+  resource_group_name = azurerm_resource_group.rg.name
+  lock_duration       = "PT5M"
+  max_delivery_count  = var.payment_queue_max_delivery_count
+
+  duplicate_detection_history_time_window = "PT15M"
+}
```

Comment:
Should these queues be be deployed to all environments?

Reply:
@msl8r the non premium servicebus namespaces have them in all environments but we use the staging queues only in aat